### PR TITLE
update 2 libs

### PR DESCRIPTION
**Pull request by shmish111**
Pre-submit checklist:
- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [ ] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [ ] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [ ] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -74,7 +74,7 @@ allow-newer: Win32-network:base
            , network-mux:base
            , ouroboros-network:base
            -- restricts to ghc<8.10
-           , prometheus:base
+           -- , prometheus:base
```
Comment (michaelpj) [inline cabal.project:77]: delete

**Review by michaelpj — APPROVED** [review]

**Comment by michaelpj** [comment]
Oh wait, can we ditch the `allow-newer` stuff relating to servant too?

**Comment by michaelpj** [comment]
`web-ghc` doesn't compile with the new `prometheus`, it actually needs some change.

**Comment by shmish111** [comment]
@michaelpj if I fix web-ghc then it breaks stack builds and will only work with cabal and nix.

**Comment by michaelpj** [comment]
Why? Can't you just change the version of `prometheus` specified in `stack.yaml` to match?

**Comment by shmish111** [comment]
oh I made a mistake, I think I can get this working.

**Comment by michaelpj** [comment]
GH actions being stupid, thanks for fixing things!

**Pull request merged**